test/lbp.jl

```diff
@@ -245,18 +245,22 @@ facts("Local Binary Patterns") do
 		@fact multi_block_lbp(img, 1, 1, 4, 4) --> 18
 	end
 
-	context("Descriptor") do
+	#=context("Descriptor") do
```

Comment:
@mronian I'll try to do it. By "manually calculated", do you mean that I have to calculate the LBP mask by hand for the whole image and then get the histogram of that mask?